### PR TITLE
Removes campaignActivity endpoint and Mongoose

### PR DESCRIPTION
**Pull request by aaronschachter**
#### What's this PR do?

Picking up from #1101 -- this PR removes code deprecated by https://github.com/DoSomething/gambit-conversations/pull/449, and it's quite a slice of :pizza:

#### How should this be reviewed?

Sanity check all endpoints used in production work as expected:

- `GET /broadcasts` (only used by Gambit Admin)
- `GET /broadcasts/:id`- used in prod for broadcast message requests
- `GET /contentfulEntries` (only used by Gambit Admin/Slack)
- `GET /contentfulEntries/:id` (only used for internal spot checking)
- `GET /defaultTopicTriggers` - used in prod to load Rivescript
- `GET /campaigns/:id`  - used in prod for signup message requests
- `GET /topics/:id`- used in prod for member message requests

#### Any background context you want to provide?

This PR lets us safely stop the Mongo database - but is blocked until https://github.com/DoSomething/gambit-conversations/pull/449 is deployed to production. 

#### Relevant tickets
* https://www.pivotaltracker.com/story/show/162330044
* https://www.pivotaltracker.com/story/show/161494958

#### Checklist
- [x] Documentation added for new features/changed endpoints.
- [x] Tested on staging.

### CODE REVIEW

**Comment by rapala61** [comment]
- Small typo in the `/campaign[s]/:id` test step.
- I tested this route and I'm getting a 500 error.
  - Route: `https://ds-mdata-responder-staging.herokuapp.com/v1/campaigns/5taFqUpJkck6y4CMCAs6is?apiKey=secretstuff`.
  - Error: 
    ```
    {
      "error": {
        "code": 500,
        "message": "phoenix campaignId=undefined error=Internal Server Error"
      }
    }
    ```

**Comment by aaronschachter** [comment]
Campaign id's are numeric -- that 500 error is from Phoenix per https://dosomething.slack.com/archives/C02BBP0CU/p1543853059003700?thread_ts=1543823525.001700&cid=C02BBP0CU

File: .github/README.md [hunk 1]
```diff
@@ -1,39 +1,47 @@
...
-* Querying [Phoenix](https://github.com/DoSomething/phoenix) for Campaign data used in Gambit message templates.
+Endpoint                                       | Functionality                                           
+---------------------------------------------- | --------------------------------------------------------
+`GET /v1/broadcasts` | [Retrieve broadcasts](endpoints/broadcasts.md#retrieve-broadcasts)
+`GET /v1/broadcasts/:id` | [Retrieve a broadcast](endpoints/broadcasts.md#retrieve-broadcast)
+`GET /v1/campaigns/:id` | [Retrieve a campaign](endpoints/campaigns.md#retrieve-campaign)
+`GET /v1/contentfulEntries/` | [Retrieve Contentful entries](endpoints/contentfulEntries.md#retrieve-contentful-entries)
+`GET /v1/contentfulEntries/:id` | [Retrieve a Contentful entry](endpoints/contentfulEntries.md#retrieve-contentful-entry)
+`GET /v1/defaultTopicTriggers` | [Retrieve all additional default topic triggers](endpoints/defaultTopicTriggers.md)
+`GET /v1/topics` | [Retrieve topics](endpoints/topics.md#retrieve-topics)
```
Comment (rapala61) [inline .github/README.md]: We should remove this route right? I get a `Cannot GET /v1/topics` when requesting it.

Comment (aaronschachter) [reply .github/README.md]: yes! Thanks for catching this 🦅

**Pull request merged**